.github/pull_request_template.md

```diff
@@ -16,7 +16,7 @@ Common breaking changes:
 
 Choose one:
  - [ ] The changes do not break compatibility
- - [ ] The changes potentially break compatibility
+ - [ ] The changes break compatibility and the API version has been updated if required
```

Comment:
I would prefer to leave this as is and put a link a link below to docs on what to do for breaking changes in frontend.